Sming/SmingCore/HardwareSerial.h

```diff
@@ -197,9 +197,20 @@ class HardwareSerial : public Stream
 	 *  @param  oneChar Character to write to the serial port
 	 *  @retval size_t Quantity of characters written (always 1)
 	 */
-	size_t write(uint8_t oneChar);
+	size_t write(uint8_t oneChar) override
+	{
+		return uart_write_char(uart, oneChar);
+	}
 
-	using Stream::write;
```

Comment:
This change breaks the compatibility because 
size_t write(const char *str);
size_t write(const char *buffer, size_t size);

are not available any longer. Make sure to add back `using Stream::write;`